modules/user_manual/pages/online_collaboration.adoc

```diff
@@ -0,0 +1,42 @@
+= Online Collaboration
+:collabora-online-url: https://www.collaboraoffice.com/collabora-online/
+:libreoffice-url: https://www.libreoffice.org/
+
+== Collabora Online
+
+{collabora-online-url}[Collabora Online] is a powerful {libreoffice-url}[LibreOffice]-based online office that supports all major document, spreadsheet and presentation file formats, and is integrable with ownCloud. 
+
+=== Secure View
+
+Secure View provides the ability to restrict access to documents, in _read-only_ shares, so that users can:
+
+. View a file with or without a custom watermark.
+. Print and export a file with a custom watermark.
+
+It is a way to securely distribute information, without that information being able to leave your organisation.
+
+NOTE: This feature is *not* available in the community edition.
+
+==== How to Use Secure View
+
+image::enterprise/collaboration/secure-view/collabora-online-administration.png[Configure Secure View in ownCloud Server.]
+
+To enable Secure View for an existing, or new, share:
+
+* Disable btn:[can edit]; Users will only be able to view documents.
+* Enable one or both of:
+** btn:[protect with watermarks]: Watermarked documents can be viewed through Collabora Online.
```

Comment:
Needs to be adapted to the admin doc as well

Reply:
This still needs to be adapted. There are only two new checkboxes as in this screenshot: https://github.com/owncloud/enterprise/issues/2917#issuecomment-492712169
For the user those are visible in the sharing dialog. For the admin they are visible in the Collabora Online settings to be able to set the default values.

Reply:
Is this still accurate for the admin view?

![](https://user-images.githubusercontent.com/13368647/55207039-e7a77880-5195-11e9-87a3-68c4c3681e52.png)

Reply:
No. As mentioned, the two permission checkboxes have a different wording.

Reply:
I've updated the user docs. Just need confirmation on the admin doc screenshot, above.